web/pdf_page_view.js

```diff
@@ -320,6 +331,7 @@ var PDFPageView = (function PDFPageViewClosure() {
     draw: function PDFPageView_draw() {
       if (this.renderingState !== RenderingStates.INITIAL) {
         console.error('Must be in new state before drawing');
+        return Promise.resolve(undefined);
```

Comment:
This has a little bit weird meaning, can we replace it with reject or cache the promise returned below?